apps-rendering/src/components/liveDateline.tsx

```diff
@@ -0,0 +1,62 @@
+// ----- Imports ----- //
+
+import { css, keyframes } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign } from '@guardian/libs';
+import { neutral, pxToRem, textSans } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { makeRelativeDate } from 'date';
+import { maybeRender } from 'lib';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+const livePulse = keyframes`
+    0% {opacity: 1;}
+    10% {opacity: .25;}
+    40% {opacity: 1;}
+    100% {opacity: 1;}
+`;
+
+const timestampStyles = (isDeadBlog = false): ReturnType<typeof css> => css`
```

Comment:
I think this function can return `SerializedStyles`

Reply:
What's the concern behind having an additional import? Is it a code quality/style/modularisation thing, or is it a bundle-size thing?